apps/next/src/app/components/nav/nav.scss

```diff
@@ -21,6 +21,11 @@
   padding-left: $fluid-spacing--small;
   display: flex;
   align-items: center;
+  border: none;
+
+  &:focus {
+    outline: none;
+  }
```

Comment:
Please don't remove the outline for focus - this is crucial for a11y. I can live with the default browser focus style.